src/main/java/io/dropwizard/revolver/RevolverBundle.java

```diff
@@ -247,14 +248,14 @@ private void initializeRevolver(final T configuration, final Environment environ
                     .build();
         }
         loadServiceConfiguration(revolverConfig);
-        System.out.println("***************************************************************************************************");
+        /*System.out.println("***************************************************************************************************");
```

Comment:
@phaneesh Had to comment this out since the log limit in travis is 4MB and it's filling up the space in logs